ext/webgpu/02_surface.js

```diff
@@ -0,0 +1,236 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+// @ts-check
+/// <reference path="../../core/lib.deno_core.d.ts" />
+/// <reference path="../web/internal.d.ts" />
+/// <reference path="../web/lib.deno_web.d.ts" />
+/// <reference path="./lib.deno_webgpu.d.ts" />
+
+const core = globalThis.Deno.core;
```

Comment:
Is this file executed? If not can we remove it? If it is then please rework to use `ext:core/mod.js` import.

Reply:
I'll update the import, however it is not executed. this is used for embedding cases only